Sources/Plasma/PubUtilLib/plGImage/plPNG.cpp

```diff
@@ -226,27 +229,59 @@ bool plPNG::IWrite(plMipmap* source, hsStream* outStream)
             row_ptrs[i] = (png_bytep)srcp + (i * stride);
         }
 
+        // Write image data
         png_write_image(png_ptr, row_ptrs);
+
+        // Prepare Textual Metadata
+        bool owned_map = false;
+        if (!textFields) {
+            auto textFields = new std::multimap<plString, plString>();
```

Comment:
Come to think of it, this is also a bug, since the inner `textFields` declaration hides the outer one, and them promptly leaks.